apps/pkcs12.c

```diff
@@ -240,6 +241,10 @@ int pkcs12_main(int argc, char **argv)
             macalg = opt_arg();
             break;
         case OPT_NODES:
+            BIO_printf(bio_err,
+                       "Warning: -nodes is deprecated; better use -noenc\n");
+            /* fall through */
```

Comment:
I do not think this is needed. Do we have other examples where we explicitly warn with such deprecated renamed options?

Reply:
I did not find any such examples.
So removed the deprecated option and related warning.